src/components/UI/InputUnderline.vue

```diff
@@ -15,9 +14,9 @@ export default defineComponent({
 
 <style lang="scss" module>
 .underline {
-  width: 100%;
-  height: 2px;
-  background-color: #7c6c4d;
+  width: 277px;
```

Comment:
```suggestion
  width: 100％;
```
widthとかは100%でいいです
どのへんをあわせるかっていうのは感覚ですが「画面幅に応じてサイズが変わってほしい」ものは固定値はやめたいです